blockly/src/utils/BlocklyCommands.java

```diff
@@ -54,6 +55,13 @@ public static void move() {
     Entity hero = Game.hero().orElseThrow(MissingHeroException::new);
     Direction viewDirection = EntityUtils.getViewDirection(hero);
     BlocklyCommands.move(viewDirection, hero);
+    Game.allEntities()
+        .filter(entity -> entity.name().equals("Blockly Black Knight"))
+        .findFirst()
+        .ifPresent(
+            boss ->
+                boss.fetch(PositionComponent.class)
+                    .ifPresent(pc -> BlocklyCommands.move(pc.viewDirection(), boss)));
```

Comment:
Eigentlich brauchen wir das nur für Level 022, aber da der Boss sonst eine maxSpeed von 0 hat, ist es in den anderen level egal, dass er versucht die schritte des helden zu kopieren. 
So richtig sexy ist diese lösung aber auch nicht.